src/util.ts

```diff
@@ -120,6 +120,19 @@ export const configLogger = () => {
   });
 };
 
+export const readGithubAppPrivateKey = (
+  path: string | undefined,
+  content: string | undefined,
+): string => {
+  if (path !== undefined) {
+    return fs.readFileSync(path, "utf-8");
+  }
+  if (content !== undefined) {
+    return content;
+  }
+  throw new Error("invalid args. path and content are undefined");
```

Comment:
[IMO] When both values are specified, it is not clear which value will be used. It would be better to provide a log of where the value was read from.